minimal-go-app/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM golang:1.16 AS build
+WORKDIR /go/src/github.com/PhilomathesInc/demo-applications/minimal-go-app
```

Comment:
Just curious, do we need to set the path under GOPATH?
Could we just create a directory like `/apps/minimal-go-app` and move the code inside it?

Reply:
@mugdha-adhav We can put it anywhere, since it is using Go modules. This is a multi stage build, and finally the binary is the only thing that that gets added in the final image.